ucrop/src/main/java/com/yalantis/ucrop/task/BitmapCropTask.java

```diff
@@ -204,7 +204,7 @@ private void copyExifForOutputFile(Context context) throws IOException {
         }
     }
 
-    private void saveImage(@NonNull Bitmap croppedBitmap) {
+    private void saveImage(@NonNull Bitmap croppedBitmap) throws IOException {
```

Comment:
Your PR description does not match the changes you have made, would you like to elaborate on the issue?